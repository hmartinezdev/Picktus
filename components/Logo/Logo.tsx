import React, { Component } from 'react';
import colors from '../../constants/colors';

class Logo extends Component {
  render() {
    return (
      <div className="logoContainer">
        <img className="logo" src="/static/logo.png" />
        <h1 className="title">Picktus</h1>
        <style jsx>{`
          .title {
            font-family: 'Caveat', cursive;
            font-size: 10rem;
            margin-left: 1rem;
            color: ${colors.secondary};
          }

          .logoContainer {
            display: flex;
            align-items: center;
            justify-content: center;
            padding: 1rem;
          }
          .logo {
            width: 200px;
          }
        `}</style>
      </div>
    );
  }
}

export default Logo;
