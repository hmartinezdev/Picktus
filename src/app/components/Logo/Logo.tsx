import Picktus from '@assets/svg/picktus.svg';
import colors from '@constants/colors';
import React, { Component } from 'react';

class Logo extends Component {
  public render(): React.ReactElement<Logo> {
    return (
      <div className="logoContainer">
        <Picktus className="logo" />
        <h1 className="title">Picktus</h1>
        <style jsx>{`
          .title {
            margin-left: 1rem;
            color: ${colors.secondary};
            font-weight: 200;
            vertical-align: center;
            position: relative;
            top: 0.7rem;
            font-family: 'Caveat', cursive;
            font-size: 5rem;
            line-height: 4rem;
            font-family: 'Josefin Sans', sans-serif;
          }

          .logoContainer {
            display: flex;
            align-items: center;
            justify-content: center;
            padding: 1.7rem 1rem 1.7rem 0.8rem;
            box-sizing: border-box;
          }

          .logo {
            width: 50%;
            height: 50%;
          }
        `}</style>
      </div>
    );
  }
}

export default Logo;
