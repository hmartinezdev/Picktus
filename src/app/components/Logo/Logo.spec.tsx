import React from 'react';
import { shallow } from 'enzyme';
import Logo from './Logo';

let wrapper;

const setup = () => shallow(<Logo />);

describe('<Logo />', () => {
  test('should render properly', () => {
    wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
