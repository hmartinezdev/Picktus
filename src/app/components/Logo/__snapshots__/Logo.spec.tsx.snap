// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should render properly 1`] = `
<div
  className="logoContainer"
>
  <img
    className="logo"
    src="/static/logo.png"
  />
  <h1
    className="title"
  >
    Picktus
  </h1>
  <style
    jsx={true}
  >
    
          .title {
            margin-left: 1rem;
            color: #FAC9B8;
            font-weight: 200;
            vertical-align: center;
            position: relative;
            top: 1rem;
            font-family: 'Caveat', cursive;
            font-size: 6.5rem;
            line-height: 8rem;
            font-family: 'Josefin Sans', sans-serif;
          }

          .logoContainer {
            display: flex;
            align-items: center;
            justify-content: center;
            padding: 1rem;
            box-sizing: border-box;
          }

          .logo {
            max-width: 130px;
          }
        
  </style>
</div>
`;
